front/app/utils/configs/participationMethodConfig/index.tsx

```diff
@@ -226,9 +225,7 @@ const informationConfig: ParticipationMethodConfig = {
   postType: 'defaultInput',
   showInputManager: false,
   isMethodLocked: false,
-  renderCTABar: (props: CTABarProps) => {
```

Comment:
It doesn't make sense to render this. We already have a "Show events" CTA button here. And it's strange to have this in the bar as the main CTA for an info phase.

Reply:
This was a decision by Rob to include it on information phases, because information phases with events are often used in lieu of an "Event" phase for projects. Events within an information phase are the main form of participation for that phase.

Reply:
Hmm, okay. I can put it back. In any case, I forgot to check with anyone from product on this. So thanks for letting me know.